milvus/HttpClient.ts

```diff
@@ -56,18 +84,24 @@ export class HttpBaseClient {
     };
   }
 
+  // fetch
+  get fetch() {
+    return this.config.fetch || fetch;
+  }
+
+  // POST API
   async POST<T>(url: string, data: Record<string, any> = {}): Promise<T> {
     try {
       // timeout controller
       const controller = new AbortController();
```

Comment:
Its also a good idea to have an options parameter to allow for `AbortController` to be passed in or use a default if not.  
```
options: {
  abortController: AbortController;
  timeout: number
};

const timeout = options?.timeout ?? this.timeout;
const abortController = options?.abortController ?? new AbortController
```

this would allow clients to have increasing amount of control.  It'll also allow easy flexibility in future s the mivlus http client gets more powerful.